registration.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-    require_once('__DIR__/config.php');
+    require_once(__DIR__ . '/config.php');
```

Comment:
Oh, I missed that change. I don't think there's any point in using `__DIR__` here (for includes in general) but of course there's also nothing wrong with it (technically).